Add more documentation for paperless

Tooa opened the pull request:
### Changes

* Add more ENV vars
* Set default for data directory
* Add more documentation
* Link install instructions from unRaid forum

> Thanks to @ljm42 as this is [original his work](https://github.com/ljm42/unRAID-CA-templates/blob/master/templates/paperless.xml)! I just did the review and thought it would be simpler if I just propose the PR. 

### Conducted Tests

* I added my fork of this repo to the docker templates of my unRaid Server and started my productive paperless instance with the changes proposed here. Everything looks fine.

Roxedus commented on templates/paperless.xml:
```diff
@@ -50,29 +54,53 @@ RUN: ./manage.py createsuperuser in the container terminal to create the superus
   <Environment>
     <Variable>
-      <Value/>
+      <Value>eng</Value>
```
Unnecessary deceleration when paired with config

Roxedus commented on templates/paperless.xml:
```diff
@@ -50,29 +54,53 @@ RUN: ./manage.py createsuperuser in the container terminal to create the superus
-  <Config Name="PAPERLESS_OCR_LANGUAGES" Target="PAPERLESS_OCR_LANGUAGES" Default="" Mode="" Description="Container Variable: PAPERLESS_OCR_LANGUAGES" Type="Variable" Display="always" Required="false" Mask="false"/>
-  <Config Name="PUID" Target="USERMAP_UID" Default="99" Mode="" Description="Container Variable: USERMAP_UID" Type="Variable" Display="always" Required="false" Mask="false">99</Config>
+  <Config Name="Port" Target="8000" Default="8000" Mode="tcp" Description="Container Port: 8000" Type="Port" Display="always" Required="false" Mask="false">8000</Config>
+  <Config Name="Data" Target="/usr/src/paperless/data" Default="/mnt/user/appdata/paperless/data" Mode="rw" Description="Container Path: /usr/src/paperless/data . &#13;&#10;This contains the paperless database. Should be in appdata." Type="Path" Display="always" Required="true" Mask="false">/mnt/user/appdata/paperless/data</Config>
```
"/mnt/user/appdata/paperless/data"
Already defined with the Default atribute

Roxedus commented on templates/paperless.xml:
```diff
@@ -4,38 +4,42 @@
-More variables here: https://github.com/the-paperless-project/paperless/blob/master/paperless.conf.example&#xD;
+  <CPUset/>
+  <DonateText/>
+  <DonateLink/>
```
No need for empty tags.

Tooa commented:
Could you take a look at my review changes? I could need some more explanations as this is my first unRaid template. What I changed:

* Removed `value` from <Config>value</Config> where the attribute `Default` is set
* Removed `value` from <Value>value</Value> in `Environment` where either (1) the attribute `Default` or (2) `value` in `<Config>value</Config>` is defined
* Removed empty tags

I have a few open questions: 

* Should we remove `rw` from  `<Mode>rw</Mode>` in the `<Volume>` as this is already defined in `<Config>`?
* Is it even possible to remove `tcp` from  `<Protocol>tcp</Protocol>`  in `<Networking>` as it is somehow defined in `<Config>` i.e. `<Config Name="Port" ... Mode="tcp">`. However, value names `mode` and `protocol` differ. Is there a mapping?

--- 

Ping me, once everything is okay on your side. I will then squash the commits and remove `WIP`.

Roxedus commented:
I have left the mode in. It doesnt hurt. This is how i would redo the current one. 
https://gist.github.com/Roxedus/53a4ff13ac0f60740435d8fc2716036b

Tooa commented:
Okay, I think it should now match your requirements. Can you take a look?

ljm42 commented:
Thanks, I think these changes (along with the instructional post in the forum) will help people get up and running faster.

One change, we don't need both PAPERLESS_OCR_LANGUAGE and PAPERLESS_OCR_LANGUAGES. According to:
  https://github.com/the-paperless-project/paperless/blob/master/paperless.conf.example
PAPERLESS_OCR_LANGUAGE is the correct one.
